docker/run.sh

```diff
@@ -47,6 +47,11 @@ if [ ! -f /var/www/html/wp-config.php ]; then
     wp --allow-root config set WP_DEBUG_DISPLAY false --raw --type=constant
 
     # Respecting Dockerfile-forwarded environment variables
+    # Allow to be reverse-proxied from https
+	wp --allow-root config set "_SERVER['HTTPS']" "isset( \$_SERVER['HTTP_X_FORWARDED_PROTO'] ) && \$_SERVER['HTTP_X_FORWARDED_PROTO'] === 'https' ? 'on' : \$_SERVER['HTTPS']" \
```

Comment:
wp-config ends up having this:

```php
$_SERVER['HTTPS'] = isset( $_SERVER['HTTP_X_FORWARDED_PROTO'] ) && $_SERVER['HTTP_X_FORWARDED_PROTO'] === 'https' ? 'on' : $_SERVER['HTTPS'];
```

Neither `$_SERVER['HTTP_X_FORWARDED_PROTO']` or `$_SERVER['HTTPS']` are set when using `wp-cli` and thus this will throw:

```
Notice: Undefined index: HTTPS
```

I had the same happen below when setting `DOCKER_REQUEST_URL` as well.

Reply:
Perhaps just create `DOCKER_REQUEST_PROTOCOL = https|http` and use it directly in `DOCKER_REQUEST_URL`?

Reply:
Fixed in dd2e4bcdbd2abb995528aeac2b656d27ab9c9161